index.html

```diff
@@ -45,6 +45,113 @@ <h1>
         <li><a href="https://instagram.com/lugardagu"><img src="header-section\peace.svg" alt="peace"></a></li> 
       </ul>                               
     </section>
+    <section class="works">
+      <div class="grid-container">
+        <div class="grid-item-flex">
+          <img src="works-section/project1-img.svg" class="w-image1" alt="image1">
+          <h2 class="toni">Tonic</h2>
+          <h6 class="history"><span class="canopy">CANOPY</span><span class="spa">&#x2022;Back End Dev</span><span class="date">&#x2022;2015</span></h6> 
+          <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul id="lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button" class="see-pro"><bold>See Project</bold></button>
```

Comment:
- [ ] Kindly make sure all these buttons have `Active`, `Hover` and `disable` styles. see design in `Figma`